README.md

```diff
@@ -82,7 +81,7 @@ that should be
 
 ## Run with Docker
 
-Please refer to [the Docker README](./docker/README.md) for instructions to run `ziti-edge-tunnel` with Docker.
+Please refer to [the container README](https://docs.openziti.io/docs/reference/tunnelers/linux/container/) for instructions to run `ziti-edge-tunnel` with Docker.
```

Comment:
can we come up with a better way than this that works for both the doc site and someone that checks the repo out?

Reply:
Currently, the ./docker/README.md file in this rep (tunneler SDK) is copied into the Docusaurus build at the link target in this PR.